server/src/main/java/org/elasticsearch/cluster/metadata/ComponentTemplateMetadata.java

```diff
@@ -103,7 +103,7 @@ public static ComponentTemplateMetadata fromXContent(XContentParser parser) thro
 
     @Override
     public Iterator<? extends ToXContent> toXContentChunked(ToXContent.Params ignored) {
```

Comment:
Nit: these arguments are no longer `ignored`, I'd rather we renamed them all back to `params`.